src/Compilers/CSharp/Portable/Binder/Semantics/OverloadResolution/OverloadResolution.cs

```diff
@@ -1748,11 +1736,13 @@ private BetterResult BetterConversionFromExpression(BoundExpression node, TypeSy
         {
             okToDowngradeToNeither = false;
 
-            if (considerRefKinds)
+            var argumentKind = node.Kind;
+            if (considerRefKinds || argumentKind == BoundKind.OutVariablePendingInference || argumentKind == BoundKind.OutDeconstructVarPendingInference)
```

Comment:
There is an open issue for the spec/implementation misalignment in #14177 and an open discussion in the LDM. When it is settled I will implement and document the new spec.

---
In reply to: [80795408](https://github.com/dotnet/roslyn/pull/13999#discussion_r80795408) [](ancestors = 80795408,80794118,80793171,80791006)